src/util/buildScript.2017071.js

```diff
@@ -62,6 +72,7 @@ function buildNewRxPatient(doc, patientResource) {
 function buildNewRxPrescriber(doc, practitionerResource) {
```

Comment:
```suggestion
function getNationalProviderIdentifier(practitionerResource) {
  for (let i = 0; i < practitionerResource.identifier.length; i++) {
    let id = practitionerResource.identifier[i];
    if (id.system && id.system.includes('us-npi')) {
      return id.value;
    }
  }
  return null;
}

function buildNewRxPrescriber(doc, practitionerResource, npi) {
```
adding a function above to separate the getting of the NPI and creation of the NewRx Prescriber tag